js/util/createIcon.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function (icon) {
```

Comment:
This function is small enough now that it might be easier to remove it and just create the icons directly.